velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -193,10 +193,12 @@ WindowFuzzer::generateSortingKeysAndOrders(
     std::vector<TypePtr>& types) {
   auto keys = generateSortingKeys(prefix, names, types);
   std::vector<SortingKeyAndOrder> results;
-  // TODO: allow randomly generating orders.
   for (auto i = 0; i < keys.size(); ++i) {
-    std::string order = "asc";
-    std::string nullsOrder = "nulls last";
+    auto asc = boost::random::uniform_int_distribution<uint32_t>(0, 1)(rng_);
```

Comment:
Hi @pramodsatya, there is a VectorFuzzer::coinToss() method. Let's use that to replace boost::random::uniform_int_distribution to make the code simpler.

Reply:
Updated, thanks. Could you please help take another look?